.vscode/launch.json

```diff
@@ -51,9 +52,15 @@
                                 "resource-server",
                                 "--log-level=debug",
                                 "--cloud-id=6575154c-72fc-4ed8-9a87-a81885ab38bb",
-                                "--backend-url=${env:BACKEND_URL}",
-                                "--backend-token=${env:BACKEND_TOKEN}"
-                        ]
+                                "--backend-url=https://search-api-open-cluster-management.apps.lab.karmalabs.corp",
```

Comment:
nit: remember to remove url/token

Reply:
The resource server will still need these parameters. Only the O2IMS CR will have them optional.
I will cleanup this file though.